app/Module/RelationshipsChartModule.php

```diff
@@ -323,6 +323,7 @@ public function chart(Individual $individual1, Individual $individual2, int $rec
             $language = $this->module_service
                 ->findByInterface(ModuleLanguageInterface::class, true)
                 ->first(fn (ModuleLanguageInterface $language): bool => $language->locale()->languageTag() === I18N::languageTag());
+            assert($language instanceof ModuleLanguageInterface, 'The application guarantees this, but static code analysis cannot infer it');
```

Comment:
Looking into this, we can obtain `$language` much more easily: `I18N::language()`